docs/conf.py

```diff
@@ -62,6 +63,8 @@
     "python": ("https://docs.python.org/3", None),
 }
 
+issues_github_path = "jazzband/pip-tools"
+issues_user_uri = "https://github.com/sponsors/{user}"
```

Comment:
This is the default already, since v4: https://github.com/sloria/sphinx-issues?tab=readme-ov-file#400-2024-01-19

```suggestion
```

Reply:
Yeah, I specifically suggested it upstream to reduce copy-paste when migrating to this plugin + spread this practice among the others..